rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/RxBleConnectionImpl.java

```diff
@@ -528,4 +533,32 @@ public BluetoothGattCharacteristic call(byte[] bytes) {
     public Observable<Integer> readRssi() {
         return rxBleRadio.queue(new RxBleRadioOperationReadRssi(gattCallback, bluetoothGatt, timeoutScheduler));
     }
+
+    @Override
+    public <T> Observable<T> queue(final RxBleRadioOperationCustom<T> operation) {
+        return rxBleRadio.queue(new RxBleRadioOperation<T>() {
```

Comment:
The javadoc of both `RxBleRadioOperationCustom#asObservable` and `RxBleConnection#queue` (this is not pushed yet, I've just added it). I think it will be good now.